client/src/main/java/io/pravega/keycloak/client/KeycloakAuthzClient.java

```diff
@@ -104,6 +101,56 @@ public String getRPT() {
         return token.getToken();
     }
 
+    public void setHttpMaxRetries(int retries) {
+        httpMaxRetries = retries;
+    }
+
+    public void setHttpRetriesDelayMsecs(int delay) {
+        httpRetriesDelayMsecs = delay;
+    }
+
+    /**
+     * Executes a Keycloak client request with retries.
+     * HttpResponseExceptions may occur.  If the error code contained within is BAD_REQUEST,
+     * SC_UNAUTHORIZED or SC_FORBIDDEN, the request is not retried as this is an authentication error
```

Comment:
nit: extra space before `If`. Also, `SC_FORBIDDEN` represents authorization error.

Reply:
This section no longer exists with my latest changes.